modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -118,5 +127,30 @@ def enum_zones
       logger.debug "Enumerated dns zones: #{zones}"
       zones
     end
+
+    def get_ptr_ip(zone, ip)
+      ip6 = ip.include?(':')
```

Comment:
> Use IPAddr#ipv6? instead.

I am working on all that right now; much easier now that I know I can require `ipaddr`